app/javascript/components/profile/team-tags-table.vue

```diff
@@ -133,6 +151,20 @@ export default {
     tagsNames() {
       return (this.tags.map((tag) => (tag.id)));
     },
+    filteredTeam() {
+      let team = [];
+      if (!this.beingFetched) {
+        team = this.team.all.filter((user) => (user.login.toLowerCase().includes(this.userFilter)));
+        if (this.selectedColor) {
+          team = team.filter((user) => (colorFromScore(user.score) === this.selectedColor));
+        }
+        if (this.selectedTag) {
+          team = team.filter((user) => (user.tags.map((tag) => (tag.name)).includes(this.selectedTag)));
+        }
+      }
+
+      return team;
+    },
```

Comment:
Quizás para que sea más directo de leer, pondría los casos al revés, cuando esté `this.beingFetched` tener la guard al principio y si no, está el resto, no anidado en un if:

```
filteredTeam() {
  let team = [];
  if(this.beingFetched) return [];

  team = this.team.all.filter(...)  
  ...
  return team;
}
```

Reply:
Bueena. Lo cambio!